src/ParseLiveQuery.js

```diff
@@ -201,12 +201,13 @@ const DefaultLiveQueryController = {
         subscription.on('error', (object) => {
           subscriptionWrap.emit('error', object);
         });
+
+        return subscriptionWrap;
```

Comment:
I've always been curious as to why the subscription needs to be wrapped.

I believe we could erase it all just do 

```
// Prevent crashing
subscription.on('error' () => {})
return subscription;
```

And document the Error Event (currently undocumented)

Reply:
I'll take a look.